### ISSUE TITLE
Failed to load plugin Obsidian 0.14.15 + 3.3.0

### ISSUE THREAD

**Issue by trainbuffer** [deathau/sliding-panes-obsidian#103]
Just updated to sliding panes 3.3.0 and it's failing to load using Obsidian public release 0.14.15.

Obsidian v0.14.15
Sliding Panes v3.3.0
macOS v12.4

error console looks like this:

`app.js:1 TypeError: Cannot read properties of undefined (reading 'children')
    at SlidingPanesPlugin._this.getRootSplits (plugin:sliding-panes-obsidian:613:27)
    at SlidingPanesPlugin._this.reallyEnable (plugin:sliding-panes-obsidian:372:36)
    at SlidingPanesPlugin._this.enable (plugin:sliding-panes-obsidian:358:27)
    at PluginBase._this.onLayoutReady (plugin:sliding-panes-obsidian:272:19)
    at e.<anonymous> (app.js:1:1466006)
    at app.js:1:235084
    at Object.next (app.js:1:235189)
    at s (app.js:1:233928)
(anonymous) @ app.js:1
2app.js:1`

`Uncaught TypeError: Cannot read properties of undefined (reading 'children')
    at SlidingPanesPlugin._this.getRootSplits (plugin:sliding-panes-obsidian:613:27)
    at SlidingPanesPlugin._this.handleLayoutChange (plugin:sliding-panes-obsidian:473:36)
    at e.t.tryTrigger (app.js:1:910451)
    at e.t.trigger (app.js:1:910384)
    at e.trigger (app.js:1:1489732)
    at e.onLayoutChange (app.js:1:1486391)
    at e.updateStateResult (app.js:1:1353675)
    at e.<anonymous> (app.js:1:1353092)
    at app.js:1:235084
    at Object.next (app.js:1:235189)
`

**Comment by MaksimZinovev** [comment]
Same problem 

Obsidian v0.14.15
Sliding panes v.3.3.0
MacOS BigSur 11.5.2

I tried to disable other plugins. So far no luck.

```
plugin:sliding-panes-obsidian:613 Uncaught (in promise) TypeError: Cannot read property 'children' of undefined
    at SlidingPanesPlugin._this.getRootSplits (VM149 plugin:sliding-panes-obsidian:613)
    at SlidingPanesPlugin._this.reallyEnable (VM149 plugin:sliding-panes-obsidian:372)
    at SlidingPanesPlugin._this.enable (VM149 plugin:sliding-panes-obsidian:358)
    at PluginBase._this.onLayoutReady (VM149 plugin:sliding-panes-obsidian:272)
    at e.onLayoutReady (app.js:1)
    at SlidingPanesPlugin.eval (VM149 plugin:sliding-panes-obsidian:331)
    at step (VM149 plugin:sliding-panes-obsidian:68)
    at Object.eval [as next] (VM149 plugin:sliding-panes-obsidian:49)
    at fulfilled (VM149 plugin:sliding-panes-obsidian:39)
﻿
```

**Comment by trainbuffer** [comment]
The last few versions have been buggy for me (changes for the new insider builds not working with the official release?)

I went back to v3.2.3 and it seems stable after a few hours of use.

**Comment by Sean-Ker** [comment]
+1 for this problem with the same error message as @MaksimZinovev.

Obsidian v0.14.15
Sliding panes v.3.3.0
Windows 11 Home - 22H2 Build 22621.105

Any idea what is causing this issue?

**Comment by EthanDow** [comment]
Same here. I had to disable the plugin for now, otherwise I can't open any of my notes.

Obsidian v0.14.15
Sliding panes v.3.3.0
macOS Monterery - Version 12.3.1

**Comment by Gortosch** [comment]
Same here. I had to disable the plugin for now, otherwise I can't open any of my notes.

Obsidian v0.14.15
Sliding panes v.3.3.0
Windows 11 Pro - 21H2 Build 22000.778

**Comment by keerah** [comment]
Same problem

**Comment by keerah** [comment]
> > Same problem Obsidian v 0.14.15 Sliding panes v 3.3.0 Windows 10 21H2 build 19044.1806
> 
> Try uninstalling the plugin and re installing, it works now.

This worked! Thank you very much

**Comment by trainbuffer** [comment]
This issue is fixed for me today updating Obsidian to v0.15.6 and Sliding Panes v3.3.0, so I guess I'll close this issue.
